config/example.yaml

```diff
@@ -12,7 +16,10 @@ database:
   connectString: localhost/XE
   user: hr
   password: welcome
+  poolMin: 3
+  poolMax: 30
 
-api:
-  name: staff-fee-privilege-api
-  endpointUri: https://api.oregonstate.edu/v1
+logger:
```

Comment:
Just comparing what we have to dropwizard, some questions came up. What is the default timezone for the log entries (the system time)? Can we set log archive and retention policies too? We'll need a way to delete old log files or else they'll fill up the disk indefinitely.